src/lib/pmd/CustomRuleRegistrar.ts

```diff
@@ -0,0 +1,83 @@
+import fs = require('fs');
+import path = require('path');
+import util = require('util');
+
+export const CUSTOM_RULE_REGISTER = path.join('catalogs', '.custom_paths.json');
```

Comment:
I'd rename this variable so it includes the word PATH or FILE or something like that.